This is merely what's required to get it to build, and caribou (the one
thing in Fedora that requires pyclutter) doesn't use this type, so it
should be safe.  But please do drop this patch once pyclutter upstream
catches up with new clutter. - ajax 2011-11-28

diff -up pyclutter-1.3.2/clutter/clutter-base-types.defs.jx pyclutter-1.3.2/clutter/clutter-base-types.defs
--- pyclutter-1.3.2/clutter/clutter-base-types.defs.jx	2010-10-24 19:00:14.000000000 -0400
+++ pyclutter-1.3.2/clutter/clutter-base-types.defs	2011-11-28 19:08:19.038001440 -0500
@@ -357,17 +357,6 @@
   )
 )
 
-(define-enum ShaderError
-  (in-module "Clutter")
-  (c-name "ClutterShaderError")
-  (gtype-id "CLUTTER_TYPE_SHADER_ERROR")
-  (values
-    '("no-asm" "CLUTTER_SHADER_ERROR_NO_ASM")
-    '("no-glsl" "CLUTTER_SHADER_ERROR_NO_GLSL")
-    '("compile" "CLUTTER_SHADER_ERROR_COMPILE")
-  )
-)
-
 (define-enum ScriptError
   (in-module "Clutter")
   (c-name "ClutterScriptError")
